KsApi/models/graphql/GraphComment.swift

```diff
@@ -57,13 +86,17 @@ extension GraphComment.GraphAuthor {
     case id
     case isCreator
     case name
+    case imageUrl
   }
 
   init(from decoder: Decoder) throws {
     let values = try decoder.container(keyedBy: CodingKeys.self)
+    let rawId = try values.decode(String.self, forKey: .id)
+    let decomposedRawId = decompose(id: rawId) ?? -1
 
-    self.id = try values.decode(String.self, forKey: .id)
+    self.id = decomposedRawId.description
```

Comment:
Why did we need the decomposed ID here?

Reply:
To show the "You" badge, we need a `Comment` `id` matching the `User` `id`, so I didn't see a problem decomposing on the root model, although maybe this should go into `Comment+GraphComment` conversion instead?

Reply:
Ah ok, yeah that's ok for now this will all change with Apollo 😄